packages/mux-player/src/index.ts

```diff
@@ -209,6 +220,15 @@ class MuxPlayerElement extends VideoApiElement {
   }
 
   #monitorLiveWindow() {
+    this.mediaController?.addEventListener('mediaplayrequest', (event) => {
+      if (
+        (event.target as Element)?.localName === 'media-play-button' &&
+        (this.streamType === StreamTypes.LIVE || this.streamType === StreamTypes.LL_LIVE)
+      ) {
+        seekToLive(this);
+      }
+    });
```

Comment:
this would still have a bug where once we switch themes it would require adding the mediaController listeners again. 

this pattern also separates the view and event listener, not sure that is the best way but it was the easiest to get this draft working because the play button is used in several places.

Reply:
These are all good things to have on our radar. Thanks for the callout.

Reply:
non-blocking: Per previous conversation, we'll likely want to revisit how this works.